tests/resources/with_cn_expected.json

```diff
@@ -0,0 +1,168 @@
+{
+    "county": {
+        "child_nutrition_data": {
+            "17001": {
+                "2019": {
+                    "afterschool_meals_count": 23197.0,
```

Comment:
If these values are all "counts" would it make more sense to store these as ints?

Reply:
I'm not sure. The bin values are floats, so I think it would make more sense to have the count values as floats for comparison (e.g. testing if `count <= bin_value` ).